root/etc/e-smith/templates/etc/httpd/conf.d/zz_nextcloud.conf/10base

```diff
@@ -14,7 +14,6 @@
 
     <IfModule mod_headers.c>
       Header always set Strict-Transport-Security "max-age=15552000"
-      Header always set Referrer-Policy "no-referrer"
```

Comment:
Why this line has been removed?

Reply:
Sorry for the lack of information !
Because from 14.0.4 version this warning has arised.
Nextcloud itself added the Referrer Policy in its `.htaccess` file, so a double insertion of the rule cause a warning message from the admin page of Nextcloud web app.